### ISSUE TITLE
Anti-Xray Paper

### ISSUE THREAD

**Issue by projectrpgmv** [PFCraft/Mohist#191]
whether Paper anti-xray is not implemented ? ?

**Comment by Mgazul** [comment]
Currently not implemented

**Comment by projectrpgmv** [comment]
I think it would be good if you implemented Paper Anti-Xray, it will reduce cheating player with xray resourcepack.

I've tried using Orebfuscator, but I had error with Protocollib. 
This my error log. https://pastebin.com/yKu5XSgz

Thanks for your attention.

**Comment by Dhannyjsb** [comment]
Yes, I had error log if try to destroy block with Orebfuscator plugin.

https://pastebin.com/hFV6zQsP

**Comment by Zilacon** [comment]
Honestly anti-xray for a modded server is a bad idea. Ive literally disabled it on every one of my servers that support it (which is all of them) mainly because of the sheer amount of modded ores it has to process for the anti-xray. It hits the server hard.

So unless its multi-threaded and off of the main server thread where it doesn't impede other server functions, its not going to be worth it.

**Comment by Dhannyjsb** [comment]
> Honestly anti-xray for a modded server is a bad idea. Ive literally disabled it on every one of my servers that support it (which is all of them) mainly because of the sheer amount of modded ores it has to process for the anti-xray. It hits the server hard.
> 
> So unless its multi-threaded and off of the main server thread where it doesn't impede other server functions, its not going to be worth it.

So how do you deal with X-ray user ?

**Comment by Zilacon** [comment]
You  dont. We have a plugin that tracks how many ores per minute someone breaks, if it exceeds 25 of the same type they  are alerted to staff. The plugin PRISM had a system like this, but sadly stopped being developed.

If your willing to sacrifice your TPS for xray protections thats all fine and dandy  for a few players online.
